vendor/k8s.io/kubernetes/pkg/util/validation/field/errors.go

```diff
@@ -201,9 +202,15 @@ func NewErrorTypeMatcher(t ErrorType) utilerrors.Matcher {
 
 // ToAggregate converts the ErrorList into an errors.Aggregate.
 func (list ErrorList) ToAggregate() utilerrors.Aggregate {
-	errs := make([]error, len(list))
-	for i := range list {
-		errs[i] = list[i]
+	errs := make([]error, 0, len(list))
```

Comment:
Any reason for creating this with the same length of `list`? Because if there are any duplicates the length will be different anyway.